app/routes/courts.js

```diff
@@ -30,6 +31,7 @@ module.exports = [
     method: 'POST',
     path: '/courts',
     options: {
+      auth: { scope: [scopes.admin] },
       tags: ['api'],
       notes: ['Creates a new court'],
       response: {
```

Comment:
Need to add scope to DELETE method of courts too

Reply:
Good catch, thanks